modules/installation-methods-for-different-architectures.adoc

```diff
@@ -21,7 +21,7 @@ a|Operator (preferred), Helm charts, or `roxctl` CLI (not recommended)
 | ppc64le ({ibm-power})
 |Yes
 |Yes ({ocp} version 4.12 and later)
-.2+a|Operator (preferred) or `roxctl` CLI
+.2+a|Operator is the only supported install method.
```

Comment:
I notice this is the only cell in the table using `.` punctuation / the only complete sentence description. Maybe this could be reworked as something like:
```suggestion
.2+a|Operator (the only supported install method)
```
to match the other cells.